packages/lesswrong/components/tagging/TagsTooltip.tsx

```diff
@@ -8,6 +8,55 @@ import { PopperPlacementType } from "@material-ui/core/Popper";
 import { defineStyles, useStyles } from "../hooks/useStyles";
 import { inferRedLinkTitle, useRedLinkPingbacks } from "./RedlinkTagPage";
 import { tagGetUrl } from "@/lib/collections/tags/helpers";
+import { useTagPageContext } from "./TagPageContext";
+
+const styles = defineStyles("TagsTooltip", theme => ({
+  tooltip: isFriendlyUI
+    ? {}
+    : {
+      padding: 0,
+      background: theme.palette.panelBackground.default,
+      boxShadow: theme.palette.boxShadow.lwTagHoverOver,
+    },
+  tooltipTitle: isFriendlyUI
+    ? {}
+    : {
```

Comment:
Something here has caused an increase in spacing in the EAF version. After/before:
![Screenshot 2025-02-03 at 16 05 39](https://github.com/user-attachments/assets/ba11f884-12e4-419f-8c4d-5f9eb32e8186)
![Screenshot 2025-02-03 at 16 05 55](https://github.com/user-attachments/assets/fed3c635-6229-4206-9435-322ea73f6a8d)

Reply:
Yeah, forgot the negation on `isFriendlyUI` for the `description` class.